RELEASE.md

```diff
@@ -1,13 +1,38 @@
-# _CDM Model - Modification to product condition_
+# _Product Model - Asset Refactoring: Asset, Index, Identifier_
+
+_Background_
+
+The Asset Refactoring initiative (see https://github.com/finos/common-domain-model/issues/2805) is seeking to improve the Product Model to address some long-standing issues and to ensure the continued extensibility to additional  financial products and markets.  A proposal is being developed - through a cross-industry Task Force - to implement this remodelling in the CDM.  
+
+This release includes the first tranche of changes to implement the refactored model (the first of three planned tranches in CDM 6).  It introduces some new data types and makes minor changes to others without significant impact to the Product structure itself.
 
 _What is being released?_
 
-Modified condition for `FpML_cd_30`
-- Addition of clause to allow `adjustedDate` check
+New `Asset` data type:
+- Introduce the new data type `Asset` which is defined as "something that can be owned and transferred in the financial markets". The data type is implemented using the new Rune DSL feature `choice` that is available in [Release 9.10](https://github.com/finos/rune-dsl/releases/tag/9.10.0).
+- Introduce the additional Asset data sub-type called `Instrument`, also using `choice`, defined as "a type of Asset that is issued by one party to one or more others".
+- Create the new base class `InstrumentBase` to model common attributes across all `Instrument` data types.
+- Introduce the new enumerator `AssetIdTypeEnum` to define certain identifier sources unique to Assets.
```

Comment:
Add: "As an extension of `ProductIdTypeEnum`"

Reply:
done